src/convertToObject.js

```diff
@@ -1,18 +1,24 @@
 'use strict';
 
-/**
- * Implement convertToObject function:
- *
- * Function takes a string with styles (see an example in [stylesString.js](./stylesString.js))
- * and returns an object where CSS properties are keys
- * and values are the values of related CSS properties (see an exampl in [test file](./convertToObject.test.js))
- *
- * @param {string} sourceString
- *
- * @return {object}
- */
 function convertToObject(sourceString) {
-  // write your code here
+  const stringArray = sourceString.split(';');
+
+  const keysAndMean = stringArray.map((item) => item.split(':'));
+
+  const stylesObject = {};
+
+  for (const line of keysAndMean) {
+    for (let i = 0; i < line.length; i++) {
+      const key = line[0];
+      const meaning = line[1];
+
+      if (line[i + 1] !== undefined) {
+        stylesObject[key.trim()] = meaning.trim();
+      }
+    }
+  }
```

Comment:
try to simplify this part

Reply:
This is the shortest solution I`ve come up with. If there is a shorter one, please, give me a hint)